### ISSUE TITLE
Update sourceforge page so all traffic is directed to github

### ISSUE THREAD

**Issue by crucialfelix** [supercollider/supercollider#1948]
Many people are ending up on sourceforge and downloading old releases.

All links and old pages need to be updated.

**Comment by gusano** [comment]
@crucialfelix I can have a look but I'd need to be added as an admin (sf account `elgusanorojo`).

**Comment by danstowell** [comment]
@gusano thanks, added you as admin

**Comment by bagong** [comment]
Googling for sc3-plugins also still leads to a sourcefourge page pointing to the sc3-downloads there...

**Comment by gusano** [comment]
@crucialfelix @danstowell I'm not sure how to proceed here and sf doc is a bit confusing..

What do we want exactly? I don't see a way to disable the *Download* sf button.

What about linking our github repo to the sf project so that *new* releases link appear on sf page? (but as I never tried that I dunno which link will appear, we have several on each GH release)

**Comment by danstowell** [comment]
I don't really know, sorry. I assumed you'd be editing the SF website to link (or even http redirect) people to github

**Comment by gusano** [comment]
@danstowell That was my plan but http://supercollider.sourceforge.net already redirects to GH. I am not sure you can get rid of the *project* page http://sourceforge.net/p/supercollider though.. Gonna research a bit more..

**Comment by gusano** [comment]
I added a README on sf [files](https://sourceforge.net/projects/supercollider/files/) page.
Now I'd like to move **all files and folders** there into a new folder called *outdated* or something.

Any objections?

**Comment by gusano** [comment]
I added a README on sf [files](https://sourceforge.net/projects/supercollider/files/) page.
Now I'd like to move **all files and folders** there into a new folder called *outdated* or something.

Any objections?

**Comment by gusano** [comment]
Actually we could as well remove completely the project on sourceforge..

**Comment by vivid-synth** [comment]
Note there were still 160 downloads of SC from sourceforge this week...

They seem to have gotten pretty slimy, if they allow us to I'd vote for removing the project completely:
http://arstechnica.com/information-technology/2015/06/sourceforge-locked-in-projects-of-fleeing-users-cashed-in-on-malvertising/

**Comment by brianlheim** [comment]
Ping. @gusano have you touched this recently? My vote would be to completely remove the project from SourceForge, that way there's one less confusing not-actually-homepage homepage for SuperCollider. I'd be willing to help in whatever way I can to make that happen ASAP.

**Comment by gusano** [comment]
@brianlheim it's been a while, I remember being stuck with that so yes, let's remove it!

**Comment by brianlheim** [comment]
Great! What can I do to help? I'm not sure I have access to anything on that site...

**Comment by gusano** [comment]
So if there are no voices against it in a week or so I propose that we remove the project completely from sourceforge (I already suggested that 9 months ago! ;) )

**Comment by brianlheim** [comment]
Great! I'm totally fine with that. Thank you!

**Comment by vivid-synth** [comment]
I think it's fine if we just pull the plug now -- I suggested removing in October and nobody had concerns

**Comment by snappizz** [comment]
I'm not entirely sure how SF works here, but let's make sure that no third party scoops up the project name. It's probably best to replace it with an empty project.

Also, let's make sure [the old wiki](http://supercollider.sourceforge.net/wiki/) doesn't get lost in the process. There are still a few pages on there that haven't been migrated to the new wiki.

**Comment by brianlheim** [comment]
Good idea. @gusano, is that OK with you? Just delete the project (after migration) and then put up a minimal/dummy project with a link here?

@snappizz What remains to be migrated?

**Comment by snappizz** [comment]
@brianlheim quite a bit. examples of unmigrated pages: http://supercollider.sourceforge.net/wiki/index.php/SimpleTrapset, http://supercollider.sourceforge.net/wiki/index.php/GUI_Design_project. not especially valuable pages, but let's not delete them.

we'll have to go through this manually to check: http://supercollider.sourceforge.net/wiki/index.php/Special:AllPages

**Comment by snappizz** [comment]
I think maybe the least destructive solution is to leave it alone and just take down the downloads.

**Comment by gusano** [comment]
@snappizz AFAIK there is no way to do that unfortunately. SF project page is not tweakable.

**Comment by muellmusik** [comment]
Is there a solution here? If not shall we close the issue?

**Comment by JordanHendersonMusic** [comment]
SF page is still there and the download is for an 8 year old version. The links above to the old wiki are dead. 

It now says there have been 4 downloads this week. 

IMH, the sf page should be removed.
